src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseStatistics.cpp

```diff
@@ -347,9 +347,21 @@ void CompositionalMultiphaseStatistics::computeRegionStatistics( MeshLevel & mes
       }
     }
     regionStatistics.averagePressure = MpiWrapper::sum( regionStatistics.averagePressure );
-    regionStatistics.averagePressure /= regionStatistics.totalUncompactedPoreVolume;
     regionStatistics.averageTemperature = MpiWrapper::sum( regionStatistics.averageTemperature );
-    regionStatistics.averageTemperature /= regionStatistics.totalUncompactedPoreVolume;
+    if( regionStatistics.totalUncompactedPoreVolume > 0 )
+    {
+      float invTotalUncompactedPoreVolume = 1.0 / regionStatistics.totalUncompactedPoreVolume;
+      regionStatistics.averagePressure *= invTotalUncompactedPoreVolume;
+      regionStatistics.averageTemperature *= invTotalUncompactedPoreVolume;
+    }
+    else
+    {
+      regionStatistics.averagePressure = 0.0;
+      regionStatistics.averageTemperature = 0.0;
+      GEOS_LOG_LEVEL_RANK_0( 1, getName() << ", " << regionNames[i]
```

Comment:
averagePressure and averageTemperature are not 0.0.  They are undetermined. Is there a constant somewhere that says "undetermined" ? Should maybe be discussed yet before making progress towards 2d outputs ?

Reply:
`NaN` ?

Reply:
It could be dangerous.
What do you think @dkachuma ?